modules/tinymce/src/plugins/template/test/ts/browser/DatesTest.ts

```diff
@@ -91,4 +91,50 @@ describe('browser.tinymce.plugins.template.DatesTest', () => {
       '</div>'
     ].join('\n'));
   });
+
+  it('TBA: Multiple replacement classes provided via options', async () => {
```

Comment:
This is intentionally using `TBA` as I just added a test for this existing functionality to make sure my change to `FilterContent` and `Templates` didn't break it as there were no tests.